### PR TITLE
Math Magicians Components

### PR DESCRIPTION
**Pull request by tresorsawasawa**
Below is a list of tasks I completed in this PR:

- [ ] Delete all the boilerplate from CRA (text, images, styles).
- [ ] Create a directory called `components`.
- [ ] Inside `components` I created a new `Calculator.js` file.
- [ ] In `Calculator.js`, I created a React component that matches [this design](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/images/calculator.png)
- [ ] Use class based components.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,9 @@
...
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => (
+  <div>
+    <Calculator />
+  </div>
+);
```
Comment (andersonlebon) [inline src/App.js]: - [x] Nice job :clap: :100: on making the `calculator.js` a class-based component, however, this main component in (`app.js`) also need to be a class-based component. Kindly let's go a head and make this App component a class-based component

File: README.md [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
+# Math magicians
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-![math-magicians_]()
+![math-magicians_](./src/images/math_magicians.png)
 
 ## Built With
 
 This project is build with:
```
Comment (andersonlebon) [inline README.md]: ## [OPTIONAL] 

- [ ] kindly you have a typing error here. write instead `This project is built with:`

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @tresorsawasawa 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tresorsawasawa** [comment]
Thank You for the review :pray:

**Review by rustam-tolipov — APPROVED** [review]
## Hi @tresorsawasawa  👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**